integration/test/agent_util.go

```diff
@@ -93,3 +97,30 @@ func ReplaceLocalStackHostName(pathIn string) {
 		log.Fatal(fmt.Sprint(err) + string(out))
 	}
 }
+
+func GetInstanceId() string {
+	// this uses the {instance_id} placeholder in the agent configuration,
+	// so we need to determine the host's instance ID for validation
```

Comment:
I think the context in the CWL integration tests gets lost by moving this comment over. Can you move this comment block back to where it was? I am fine with pulling the code here as a function in this util file, just don't think the comment here is 100% relevant generally - it's referencing the `{instance_id}` placeholder in log configuration.

Reply:
It makes sense in both context since we need the instance id for validation for both tests. I will still move.

Reply:
Right. We _use_ the instance ID get we fetch from IMDS in both tests, but I think the comment in question is misleading because it specifically points out the log configuration, which is not relevant to using the `{aws:InstanceId}` aggregation dimension, but that's just me being super nitpicky about it